monitoring/Cargo.toml

```diff
@@ -22,3 +22,4 @@ slog_derive = "0.1.1"
 networking = { path = "../networking" }
 tezos_messages = { path = "../tezos/messages" }
 tezos_encoding = { path = "../tezos/encoding" }
+rand = ""
```

Comment:
We should be using at least a major verion number.

Reply:
Fixed in 8acf122